amundsen_common/models/table.py

```diff
@@ -106,6 +106,18 @@ def default_if_none(arg: Optional[bool]) -> bool:
     return arg or False
 
 
+@attr.s(auto_attribs=True, kw_only=True)
+class ProgrammaticDescription:
+    source_id: str
```

Comment:
Shall we name it `source` to unify with `databuilder`?

Reply:
thanks!